### PR TITLE
zh-translation: /news/releases/0.x/announcing-0.8/index.md #1526

### PR DESCRIPTION
**Pull request by gorda**
Ref: [ServiceMesher#1526](https://github.com/servicemesher/istio-official-translation/issues/1526)

### CODE REVIEW

File: content/zh/news/releases/0.x/announcing-0.8/index.md [hunk 1]
```diff
@@ -12,52 +12,44 @@ aliases:
...
-[new traffic management APIs](/zh/blog/2018/v1alpha3-routing/). We believe this new model is easier to understand while covering more real world
-deployment [use-cases](/zh/docs/tasks/traffic-management/). For folks upgrading from earlier releases there is a
-[migration guide](/zh/docs/setup/upgrade/) and a conversion tool built into `istioctl` to help convert your configuration from the old model.
+- **改良的流量管理模型**。我们终于准备好完成我们的[新流量管理 API](/zh/blog/2018/v1alpha3-routing/) 的总结。 我们相信，在涵盖更多实际部署[用例](/zh/docs/tasks/traffic-management/)的同时，这种新模型更易于理解。 对于从早期发行版升级的人，这儿有一个[迁移指南](/zh/docs/setup/upgrade/)和内置在 `istioctl` 中的转换工具，可帮助您从旧模型转换配置。
 
-- **Streaming Envoy configuration**. By default Pilot now streams configuration to Envoy using its [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst). This new approach increases effective scalability, reduces rollout delay and should eliminate spurious 404 errors.
+- **流 Envoy 配置**。默认情况下，Pilot 现在使用其 [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst) 将配置流式传输到 Envoy。 这种新方法提高了有效的可伸缩性，减少了推出延迟，应该能消除虚假的 404 错误。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.8/index.md]:
```suggestion
- **Envoy 流式配置**。默认情况下，Pilot 现在使用其 [ADS API](https://github.com/envoyproxy/data-plane-api/blob/master/xds_protocol.rst) 将配置流式传输到 Envoy。 这种新方法提高了有效的可伸缩性，减少了推出延迟，应该能消除虚假的 404 错误。
```

File: content/zh/news/releases/0.x/announcing-0.8/index.md [hunk 2]
```diff
@@ -12,52 +12,44 @@ aliases:
...
+- **Ingress/Egress 的 Gateway**。我们不再支持将 Kubernetes Ingress 规范与 Istio 路由规则结合使用，因为它导致了许多错误和可靠性问题。Istio 现在支持用于 ingress 和 egress 代理的独立于平台的 [Gateway](/zh/docs/concepts/traffic-management/#gateways) 模型，该模型可跨 Kubernetes 和 Cloud Foundry 使用，并与路由无缝协作。Gateway 支持基于[服务器名称指示](https://en.wikipedia.org/wiki/Server_Name_Indication)的路由，并基于客户端提供的服务器名称提供证书。
 
-- **Constrained Inbound Ports**. We now restrict the inbound ports in a pod to the ones declared by the apps running inside that pod.
+- **受限的入站端口**。现在，我们将 Pod 中的入站端口限制为该 Pod 中运行的应用所声明的端口。
 
-## Security
+## 安全{#security}
 
-- **Introducing Citadel**. We've finally given a name to our security component. What was formerly known as Istio-Auth or Istio-CA is now called Citadel.
+- **Citadel 介绍**。我们终于给安全组件起了个名字。以前的 Istio-Auth 或者 Istio-CA 的现在统称为 Citadel。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.8/index.md]:
```suggestion
- **Citadel 介绍**。我们终于给安全组件起了个名字。以前的 Istio-Auth 或者 Istio-CA 现在被统称为 Citadel。
```

File: content/zh/news/releases/0.x/announcing-0.8/index.md [hunk 3]
```diff
@@ -12,52 +12,44 @@ aliases:
...
 
-- **CloudWatch**. Mixer can now report metrics to AWS CloudWatch.
-[Learn more](/zh/docs/reference/config/policy-and-telemetry/adapters/cloudwatch/)
+- **CloudWatch**。Mixer 现在可以将指标报告给 AWS CloudWatch。[了解更多](/zh/docs/reference/config/policy-and-telemetry/adapters/cloudwatch/)
 
-## Known issues with 0.8
+## 0.8 已知问题{#known-issues-with-0.8}
 
-- A gateway with virtual services pointing to a headless service won't work ([Issue #5005](https://github.com/istio/istio/issues/5005)).
+- 具有指向 headless 服务的虚拟服务的网关无法工作（[Issue #5005](https://github.com/istio/istio/issues/5005)）。
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.8/index.md]:
```suggestion
- 有关指向 headless 服务的虚拟服务网关无法工作的（[Issue #5005](https://github.com/istio/istio/issues/5005)）。
```

File: content/zh/news/releases/0.x/announcing-0.8/index.md [hunk 4]
```diff
@@ -12,52 +12,44 @@ aliases:
...
 
-- There is a [problem with Google Kubernetes Engine 1.10.2](https://github.com/istio/istio/issues/5723). The workaround is to use Kubernetes 1.9 or switch the node image to Ubuntu. A fix is expected in GKE 1.10.4.
+- 这是一个 [Google Kubernetes Engine 1.10.2 的问题](https://github.com/istio/istio/issues/5723)。变通的方法是使用 Kubernetes 1.9 或者将节点的镜像切换为 Ubuntu。该问题预计在 GKE 1.10.4 会得到修复。
 
-- There is a known namespace issue with `istioctl experimental convert-networking-config` tool where the desired namespace may be changed to the `istio-system` namespace, please manually adjust to use the desired namespace after running the conversation tool.   [Learn more](https://github.com/istio/istio/issues/5817)
+- `istioctl experimental convert-networking-config` 工具存在一个已知的命名空间问题，所需的命名空间可能会被修改为 `istio-system`，请在运行对话工具后手动将其修改为所需的名称空间。[了解更多](https://github.com/istio/istio/issues/5817)
```
Comment (yuxiaobo96) [inline content/zh/news/releases/0.x/announcing-0.8/index.md]:
```suggestion
- `istioctl experimental convert-networking-config` 工具存在一个已知的命名空间问题，所需的命名空间可能会被修改为 `istio-system`，请在运行对话工具后手动将其修改为所需的命名空间。[了解更多](https://github.com/istio/istio/issues/5817)
```

Comment (gorda) [reply content/zh/news/releases/0.x/announcing-0.8/index.md]: @yuxiaobo96 已修改，非常感谢细心 review

image

**Pull request merged**